frontend/src/lib/components/PayGateMini/payGateMiniLogic.tsx

```diff
@@ -97,7 +97,7 @@ export const payGateMiniLogic = kea<payGateMiniLogicType>([
             (s) => [s.productWithFeature],
             (productWithFeature) => {
                 const currentPlanIndex = productWithFeature?.plans.findIndex((plan) => plan.current_plan)
-                if (!currentPlanIndex) {
+                if (currentPlanIndex === undefined || currentPlanIndex < 0) {
```

Comment:
This was a bug if you were on the free plan because `!currentPlanIndex` is true if 0